parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -209,6 +223,39 @@ ensureAPMZ() {
     ln -Ffs "$bin_path" "$apmz_filepath" # symlink apmz into /usr/local/bin/apmz
 }
 
+# install mostly-static bpftrace binary linked only against glibc > 2.23
```

Comment:
I believe we're tight on cloud-init space and we generally try to avoid comments that aren't critical in this file, @jackfrancis to confirm

Reply:
ironically the answer to your other comment is this line 😅 but understandable, will remove

Reply:
i'll add back if jack negatively confirms